support/rmt

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+#############################################################
+# Name:        Supportconfig Plugin for RMT
+# Description: Gathers important troubleshooting information
+#              about RMT
+# License:     GPLv2
+# Author:      SCC Team <happy-customer@suse.de>
+# Modified:    2018 April 18
+#############################################################
+
+SVER=0.0.1
+RCFILE="/usr/lib/supportconfig/resources/scplugin.rc"
+
+[ -s $RCFILE ] && . $RCFILE || { echo "ERROR: Initializing resource file: $RCFILE"; exit 1; }
+
+## Our own helper functions
+validate_rpm_if_installed() {
+  THISRPM=$1
+  echo "#==[ Validating RPM ]=================================#"
+  if rpm -q "$THISRPM" >/dev/null 2>&1; then
+    echo "# rpm -V $THISRPM"
+
+    if rpm -V "$THISRPM"; then
+      echo "Status: Passed"
+    else
+      echo "Status: WARNING"
+    fi
+  else
+    echo "package $THISRPM is not installed"
+    echo "Status: Skipped"
+  fi
+  echo
+}
+
+RPMLIST="
+rmt-server
+nginx
+mariadb
+"
+SERVICES="rmt rmt-server-sync rmt-server-mirror mariadb nginx"
+
+CONF_FILES="/etc/rmt.conf /etc/nginx/vhosts.d/rmt-server-http.conf /etc/nginx/vhosts.d/rmt-server-https.conf"
+
+section_header "Supportconfig Plugin for RMT, v${SVER}"
+if ! rpm -q rmt-server &>/dev/null; then
+  echo "ERROR: RMT rpm package ('rmt-server') not installed"
+  echo
```

Comment:
I don't know if this is confusing for people. I always try to avoid echo without arguments. Using `echo ""` or even `echo -e "..\n".